cmd/root/kubernetes/kubernetes.go

```diff
@@ -128,3 +128,12 @@ func (c *KubectlK8sActions) FindHosts(searchTerm string) (podName []string, err
 func (c *KubectlK8sActions) HelpText() string {
 	return "Make sure the labels and namespace you use actually correspond to your dremio pods: try something like 'ddc -n mynamespace --coordinator app=dremio-coordinator --executor app=dremio-executor'.  You can also run 'kubectl get pods --show-labels' to see what labels are available to use for your dremio pods"
 }
+
+func (c *KubectlK8sActions) CopyLogs(hostString string, isCoordinator bool, source, destination string) (out string, err error) {
```

Comment:
* Not sure why we're double cleaning here as the c.cleanLocal is already doing the same as the strings.Replace
* I don't think we need the CopyLogs bit at all do we?